Server/test/controller/authControllerTest.js

```diff
@@ -86,6 +85,7 @@ describe('AuthControler', function () {
                     password: "012345678",
                     role: "admin"
                 });
+            admin = res;
```

Comment:
never used

Reply:
It is but as a global. Not really pretty but it speeds up the test by 500 ms so why not